test_scripts/Defects/4_6/964_SDL_sends_to_mobile_APPLICATION_NOT_REGISTRED_in_setAppIcon_responce_if_HMI_responds_with_INVALID_DATA.lua

```diff
@@ -0,0 +1,74 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/964
+--
+-- Precondition:
+-- SDL Core and HMI are started. App is registered, HMI level = FULL
+-- Description:
+-- Steps to reproduce:
+-- 1) Mobile app sends PutFile(request).
+-- 2) Mobile app sends invalid SetAppIcon(requests).
+-- Expected:
```

Comment:
Please update.
`Respond successfully processed PutFile requests.` - who respond? maybe SDL, also maybe **to**  processed PutFile **request**